scrypto-test/src/ledger_simulator/ledger_simulator.rs

```diff
@@ -872,6 +872,20 @@ impl<E: NativeVmExtension, D: TestDatabase> LedgerSimulator<E, D> {
         reader.read_typed_kv_entry(kv_store_id.as_node_id(), key)
     }
 
+    pub fn get_resource_total_supply(&self, resource: ResourceAddress) -> Decimal {
```

Comment:
Can we rename this to `get_fungible_resource_total_supply`?

Reply:
Good day mate. Done.